### PR TITLE
Add composer analyser checker

### PR DESCRIPTION
**Pull request by Neirda24**

### CODE REVIEW

**Comment by Neirda24** [comment]
PHPStan issue is unrelated. Fixed in another PR.

**Review by Jean-Beru — APPROVED** [review]
Great, I did not know this tool👌

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -6,7 +6,9 @@
         __DIR__ . '/docs',
         __DIR__ . '/src',
         __DIR__ . '/tests',
+        __DIR__ . '/',
```
Comment (Jean-Beru) [inline .php-cs-fixer.dist.php:9]: Since the Finder is recursive, defining previous paths is useless

Comment (Neirda24) [reply .php-cs-fixer.dist.php:9]: It was to include php files at the root of the library. I hesitated. Should we include php files at the root of the library in cs fixer too ?

**Pull request merged**